public/js/post/post.js

```diff
@@ -0,0 +1,38 @@
+var CourseCode = React.createClass({
+    getInitialState: function() {
+        return {selected: false}
+    },
+
+    componentWillMount: function() {
+        var isSelected = (getCookie(this.props.courseID) == 'active') ? true : false;
```

Comment:
I imagine the `? true : false` is redundant; just return the value of the comparison. Also, use `===` not `==`.